packages/app-project/src/helpers/fetchOrganization/fetchOrganization.js

```diff
@@ -18,6 +18,7 @@ async function fetchOrganizationData(organizationID, env) {
   } catch (error) {
     logToSentry(error)
     console.log('Error loading organization:', error)
+    return null
```

Comment:
There's an ESLint rule to catch inconsistent returns from functions. It would have caught this, but I don't think it's set on the project app.

Reply:
https://github.com/zooniverse/front-end-monorepo/blob/24b9fb49ebe2c825e21dc8c593e7df301e5e08ba/packages/tools-standard/options.js#L24-L26